alfresco-transform-misc/alfresco-transform-misc-boot/src/main/java/org/alfresco/transformer/MiscController.java

```diff
@@ -1,159 +1,158 @@
-/*

```

Comment:
Format only changes should be done in separate PRs - did anything else change other than line-endings (cc @eknizat @PunchedChimera ) ?